themes/Atomize-color-theme.json

```diff
@@ -256,6 +256,7 @@
 		"statusBarItem.hoverBackground": "#2F333D",
 		"statusBarItem.prominentBackground": "#21252B",
 		"statusBarItem.prominentHoverBackground": "#2F333D",
+		"statusBarItem.remoteBackground": "#21252B",
 
```

Comment:
Do you mind making the foreground the same color as the other items on the status bar as well then?

```
"statusBarItem.remoteForeground": "#9DA5B4"
```

Reply:
The foreground uses the same color as other items by default, but I don't mind adding this line in case there are future changes.

Reply:
Oh I made a mistake. Yes, the foreground should have been #9DA5B4.